build_targets

```diff
@@ -4,4 +4,6 @@ beryllium userdebug yes yes
 guacamole userdebug yes yes
 lemonade userdebug yes yes
 lemonadep userdebug yes yes
+mars userdebug yes yes
```

Comment:
included kernel twice
https://github.com/crdroidandroid/android_device_xiaomi_mars/blob/14.0/crdroid.dependencies#L11
https://github.com/crdroidandroid/android_device_xiaomi_sm8350-common/blob/14.0/crdroid.dependencies#L3

Reply:
Kernel dependencies have been removed from the device tree